src/cmake/MinGW-CrossCompile-Windows.cmake

```diff
@@ -0,0 +1,38 @@
+# http://www.cmake.org/Wiki/CMake_Cross_Compiling
```

Comment:
File to be removed (may only be needed if CI build moves from AppVeyor to Travis)